src/include/misc.h

```diff
@@ -255,3 +255,11 @@
 #define	__wt_page_swap(session, held, want, flags)			\
 	__wt_page_swap_func(session, held, want, flags)
 #endif
+
+/* Random number generator state. */
+union __wt_rand_state {
+	uint64_t v;
+	struct {
+		uint32_t w, z;
+	} x;
+};
```

Comment:
@keithbostic, I screwed up the struct first time around when moving it into the header and wanted to avoid doing the it again in the future.  In other places we use "u" for a union: I'm trying to be consistent here because the "x" here is a struct not a union.